### PR TITLE
Admin Send Email

### PR DESCRIPTION
**Pull request by annq3012**
## Description
Send Email to Borrower in a long time and update date_send_email
## Reference Requirements
- Product Backlog ID: PBI-3
- Task ID: TI-3.2

## Reference Isuees
#{issue_id}

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)
update

### CODE REVIEW

File: app/Model/Borrowing.php [hunk 1]
```diff
@@ -61,4 +62,14 @@ public function users()
     public $sortable = ['from_date', 'to_date', 'date_sent_mail'];
     
     public $sortableAs = ['users.employee_code', 'users.name', 'users.email', 'books.name'];
+
+    /**
+     * Get Attribute Date send email
+     *
+     * @return array
+    */
+    public function getDateSendMailAttribute()
```
Comment (at-anhngo) [inline app/Model/Borrowing.php]: hàm này khai báo mà ko thấy gọi nhể?

**Review by ghost — COMMENTED** [review]
Check my comment

File: app/Model/Borrowing.php (hunk 1 above)
Comment (at-annguyen) [reply app/Model/Borrowing.php]: borrow blade php

File: resources/lang/en/borrow.php [hunk 2]
```diff
@@ -11,6 +11,9 @@
     'end_date' => 'End date',
     'title_borrow' => 'LIST OF BORROWINGS',
     'date_sent_mail' => 'Date Sent Mail',
+    'send_mail_success' => 'Send Mail Success',
+    'send_mail_failure' => 'Send Mail failure',
```
Comment (at-anhngo) [inline resources/lang/en/borrow.php]: sửa lại cho đồng bô, nếu là message thì chỉ cần viết hoa chữ cái đầu thôi

**Review by ghost — COMMENTED** [review]
Check my comment

**Comment by at-anhngo** [comment]
LGTM

File: resources/views/backend/books/borrow.blade.php [hunk 3]
```diff
@@ -43,16 +43,17 @@
                                 <td>{{ $borrowing->books->name }}</td>
                                 <td>{{ date(config('define.date_format'), strtotime($borrowing->from_date)) }}</td>
                                 <td>{{ date(config('define.date_format'), strtotime($borrowing->to_date)) }}</td>
-                                <td>{{ $borrowing->date_send_email }}</td>
+                                <td>{{ $borrowing->DateSendMail }}</td>
```
Comment (at-huydq) [inline resources/views/backend/books/borrow.blade.php]: cái cũ `$borrowing->date_send_email` đúng rồi sao sửa lại e ?

**Review by ghost — COMMENTED** [review]
check comment

**Comment by at-huydq** [comment]
LGTM

**Pull request merged**